lookout/style/format/analyzer.py

```diff
@@ -230,3 +269,11 @@ def _filter_files(cls, files: Dict[str, File], line_length_limit: int
         if excluded > 0:
             cls.log.debug("excluded %d/%d files by max line length %d",
                           excluded, total, line_length_limit)
+
+    @staticmethod
+    def recursive_update(mapping, other):
+        for key, value in other.items():
+            if isinstance(value, dict):
+                FormatAnalyzer.recursive_update(mapping.setdefault(key, {}), value)
```

Comment:
Please change the implementation from recursive to non-recursive. It ts very easy.

Reply:
https://github.com/src-d/style-analyzer/pull/200